app/data/test_skip_condition_group.json

```diff
@@ -0,0 +1,118 @@
+    "schema_version": "0.0.1",
+    "data_version": "0.0.1",
+    "survey_id": "0",
+    "title": "Skip group",
+    "theme": "default",
+    "description": "A questionnaire to skipping a group.",
+    "eq_id": "0",
+    "messages": {
+      "INTEGER_TOO_LARGE": "Number is too large",
+      "NEGATIVE_INTEGER": "Number cannot be less than zero",
+      "NOT_INTEGER": "Please enter an integer"
+    },
+    "groups": [
+        {
+            "id": "do-you-want-to-skip-group",
+            "title": "Do you want to skip the next group?",
+            "blocks": [
+                {
+                    "type": "questionnaire",
+                    "id": "do-you-want-to-skip",
+                    "routing_rules":[],
+                    "title": "",
+                    "validation": [],
+                    "sections": [
+                        {
+                            "description": "",
+                            "id": "do-you-want-to-skip-section",
+                            "title": "Do you want to skip?",
+                            "validation": [],
+                            "questions": [
+                              {
+                                    "description": "",
+                                    "id": "do-you-want-to-skip-question",
+                                    "title": "Do you want to skip?",
+                                    "type": "General",
+                                    "answers": [
+                                        {
+                                            "guidance": "",
+                                            "id": "do-you-want-to-skip-answer",
+                                            "label": "Do you want to skip?",
+                                            "mandatory": true,
+                                            "options": [
+                                                {
+                                                    "label": "Yes",
+                                                    "value": "Yes"
+                                                },
+                                                {
+                                                    "label": "No",
+                                                    "value": "No"
+                                                }
+                                            ],
+                                            "type": "Radio",
+                                            "validation": {
+                                                "messages": {}
+                                            }
+                                        }
+                                    ]
+                              }
+                            ]
+                        }
+                    ]
+                }
+            ]
+        },
+        {
+            "id": "should-skip-group",
+            "title": "This question may or may not be skipped",
+            "skip_condition": {
```

Comment:
Yep, still a bit WIP but wanted to get some views on how we are implementing this